phpstan.neon.dist

```diff
@@ -31,6 +31,27 @@ parameters:
             count: 1
             path: src/EmulatedHttpClient.php
 
+        # we still support the obsolete RequestFactory for BC but do not require the package anymore
```

Comment:
there is something with stubs one can do in phpstan but did not get it to work.

we will get rid of all this when we do a new major version anyways, so did not want to spend more time on it.